ext/bg/js/backend.js

```diff
@@ -613,11 +613,14 @@ class Backend {
                 try {
                     const tab = await Backend._findTab(1000, (url2) => (
                         url2 !== null &&
-                        url2.startsWith(url) &&
-                        (url2.length === url.length || url2[url.length] === '?' || url2[url.length] === '#')
+                        url2.startsWith(baseUrl) &&
+                        (url2.length === baseUrl.length || url2[baseUrl.length] === '?' || url2[baseUrl.length] === '#')
                     ));
                     if (tab !== null) {
                         await Backend._focusTab(tab);
+                        if (queryParams.query) {
+                            await new Promise((resolve) => chrome.tabs.update(tab.id, {url}, resolve));
+                        }
```

Comment:
Changed this in d4e74a05723b1b4347c51606bcee61dc72e34e51 because the new `mode` query parameter broke focusing an existing tab. Now it's also possible to set a new query on an existing search tab.